Library/Formula/racer.rb

```diff
@@ -0,0 +1,43 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.0.0.tar.gz"
+  sha256 "78895296ed688eeccbaf7745235f0fc503407bfa718f53583a4dcc9e1246b7f5"
+  head "https://github.com/phildawes/racer.git"
+
+  option "without-src", "Do not get rust source for racer from www.rust-lang.org"
+
+  depends_on "rust" => :build
+
+  resource "rustsrc" do
+    url "https://static.rust-lang.org/dist/rustc-1.3.0-src.tar.gz"
+    sha256 "ea02d7bc9e7de5b8be3fe6b37ea9b2bd823f9a532c8e4c47d02f37f24ffa3126"
+  end
+
+  def install
+    if build.with?("src")
+      resource("rustsrc").stage do
+        rm_rf "src/llvm"
+        rm_rf "src/test"
+        (share/"rustsrc").install Dir["./src/*"]
+      end
+    end
+
+    system "cargo", "build", "--release"
+    libexec.install "target/release/racer"
+  end
+
+  def caveats; <<-EOS.undent
+    racer in installed at #{opt_prefix}/libexec/racer, set it to your editor if needed.
+    Rust source is installed at #{opt_prefix}/share/rustsrc by default.
```

Comment:
Can you specify the version here? Perhaps using ```#{resource(rustsrc).version}``` would do it? I would word it like this:

```
You should set RUST_SRC_PATH to the Rust source code. Rust #{resource(rustsrc).version} is installed at #{opt_prefix}/share/rustsrc for your convenience. If completion for another version of Rust is required, it can be installed in any directory and RUST_SRC_PATH should be set accordingly.
```